dsl_lang/lib/q.sched

```diff
@@ -1,2 +1,6 @@
-classes CU, CV, CW, CX; 
-return rr[rr[CU, CV], strict[CW, CX]]
\ No newline at end of file
+classes B, RP, RT;
```

Comment:
Maybe this file can be removed now that we are able to test with the sample files under `dsl`?